mwdb/web/src/components/RecentView/RecentView.js

```diff
@@ -31,61 +31,91 @@ export default function RecentView(props) {
     // Query error shown under the query bar
     const [queryError, setQueryError] = useState(null);
     const [objectCount, setObjectCount] = useState(null);
-
+    // const [countingEnabled, setCountingEnabled] = useState(true);
+    const countingEnabled = searchParams.get("count") === "0" ? 0 : 1;
```

Comment:
First of all, I know I'm changing my mind right now :smile: I think the default state should be turned off counting. 

First I was thinking that we should keep the original behavior as a default, but whole feature idea originates in people claiming that search performance is poor and one of the reasons is counting. So default should be 0.